packages/components/src/components/editor/browser/browser.stories.tsx

```diff
@@ -30,7 +30,7 @@ type DefaultBrowserProps = {
 };
 
 const DefaultBrowser = ({ applyFn, applyBtn, initSearch }: DefaultBrowserProps) => {
-  const roles = useBrowser(roleData, undefined, initSearch);
+  const roles = useBrowser(roleData, undefined, initSearch, undefined, { '0.1': true });
```

Comment:
This is exactly the reason why at a specific point a option object makes sense.
So I know this means a breaking change, but I would not change this so all optional properties for this hook live in a optional object, so this gets possible
`useBrowser(roleData, {initialSearch: initSearch, initialSelected: { '0.1': true }});`